coriolis/osmorphing/osmount/base.py

```diff
@@ -107,12 +108,25 @@ def _get_pvs(self):
             if line == "":
                 continue
             line = line.strip().split(":")
+            if len(line) < PVDISPLAY_VALUES_COUNT:
```

Comment:
This is not ideal as it's theoretically possible for them to add or even remove values from there in the future.

I'd remove this assumption and just stick to checking that `len(line) >= 2`